pages/api/play-attempt/index.ts

```diff
@@ -8,6 +8,53 @@ import getTs from '../../../helpers/getTs';
 
 const MINUTE = 60;
 
+export async function forceUpdateLatestPlayAttempt(userId:string, levelId:string, context:AttemptContext, ts:number) {
+  const found = await PlayAttemptModel.findOneAndUpdate({
+    userId: userId,
+    levelId: levelId,
+  }, {
+    $set: {
+      attemptContext: context,
+      endTime: ts,
+    },
+    $inc: { updateCount: 1 }
+  }, {
+    new: false,
+    sort: { _id: -1 }
+  });
+  let sumAdd = 0;
+
+  if (found && context !== AttemptContext.BEATEN) {
+    sumAdd = ts - found.endTime;
+  }
+
+  if (sumAdd || context === AttemptContext.JUST_BEATEN) {
```

Comment:
can this logic just be put inside the if statement above? seems like if the first is true the second should always be true

Reply:
I guess it is technically possible for sumAdd to be 0…